### ISSUE TITLE
add satellite tails to /geo

### ISSUE THREAD

**Issue by jeffpc** [ahupowerdns/galmon#6]
It could possibly look very cool if for each sat on the map, there were 3 or 4 "previous positions" drawn as well.  This is a common visualization on ATC radar showing the previous radar returns to at a glance ascertain the direction of the craft.

Estimating how fast GNSS sats move based on wikipedia MEO info, drawing the previous positions at 5, 10, and 15 mins prior to "current" may work well.  The older the point, the less saturated it should be.

**Comment by nousian** [comment]
This would be pretty nice visualization. 

It would most likely need separate svn-history.json file for track history though. Maybe say 10 latest positions, saved every minute?

**Comment by jeffpc** [comment]
I don't think it needs a new json.  The map is not meant to be cm-accurate location info for the satellites, so IMO it should be enough to do a quick & dirty client-side calculation based on the inclination and current position.

**Comment by mahtin** [comment]
@jeffpc / @nousian - I'm already thinking about this and coding this as a client-side item. It will display a trail until you refresh the page.

**Comment by nousian** [comment]
I think you're right. Could be done client side only.

Speaking of visualization, there are many interesting aspects left to visualize. One I have been thinking of is TLE vs almanac vs real position, but this cannot be done on global map as the differences are mostly in kilometer range maximum.

**Comment by nousian** [comment]
@mahtin - maybe you can apply the same client-side "tails" on the observer.html graph as well?
